bigquery_etl/cli/check.py

```diff
@@ -155,18 +165,22 @@ def _render(
 
 @check.command(
     help="""
-    Run ETL checks.
-s    \b
+    Runs data checks defined for the dataset (checks.sql).
```

Comment:
```suggestion
    Runs data checks defined for the query (checks.sql).
```
The term dataset is a bit overloaded